File: scripts/query_parcel.py
import sqlite3, json, sys, os
root = os.path.dirname(os.path.dirname(__file__))
db = os.path.join(root, 'prebuilt', 'parcelapp.db')
num = sys.argv[1] if len(sys.argv) > 1 else '0522010201354'
con = sqlite3.connect(db)
con.row_factory = sqlite3.Row
cur = con.cursor()
cur.execute('SELECT id, num_parcel, parcel_type, geometry, properties FROM parcels WHERE num_parcel = ? LIMIT 1', (num,))
row = cur.fetchone()
if not row:
    print('Not found')
else:
    out = dict(row)
    try:
        out['properties'] = json.loads(out['properties'])
    except Exception:
        pass
    try:
        out['geometry'] = json.loads(out['geometry'])
    except Exception:
        pass
    print(json.dumps(out, ensure_ascii=False, indent=2))
con.close()

File: scripts/query_parcel.js
const Database = require('better-sqlite3');
const path = require('path');
const dbPath = path.resolve(__dirname, '..', 'prebuilt', 'parcelapp.db');
const db = new Database(dbPath, { readonly: true });
const num = process.argv[2] || '0522010201354';
const row = db.prepare('SELECT id, num_parcel, parcel_type, geometry, properties FROM parcels WHERE num_parcel = ? LIMIT 1').get(num);
console.log('Querying:', num);
if (!row) { console.log('Not found'); process.exit(0); }
try { row.properties = JSON.parse(row.properties); } catch (e) {}
try { row.geometry = JSON.parse(row.geometry); } catch (e) {}
console.log(JSON.stringify(row, null, 2));
db.close();
